projects/cds/src/profiletag/services/profile-tag.injector.spec.ts

```diff
@@ -37,24 +38,23 @@ const mockCDSConfig: CdsConfig = {
 
 describe('ProfileTagInjector', () => {
   let profileTagInjector: ProfileTagInjector;
-  let nativeWindow: ProfileTagWindowObject;
+  let profileTagEventTracker: ProfileTagEventTracker;
+  let spartacusEventTracker;
   let getActiveBehavior;
   let baseSiteService;
-  let appendChildSpy;
   let getConsentBehavior;
   let isConsentGivenValue;
   let routerEventsBehavior;
   let router;
   let consentsService;
-  let mockedWindowRef;
   let cartService;
   let orderEntryBehavior;
   let cartBehavior;
+
   function setVariables() {
     getActiveBehavior = new BehaviorSubject<String>('');
```

Comment:
I just noticed, the generic type should be lowercase `string` instead of `String`.